apps/builder/app/builder/features/sidebar-left/help-popover.tsx

```diff
@@ -0,0 +1,111 @@
+import {
+  Button,
+  Flex,
+  Popover,
+  PopoverContent,
+  PopoverTrigger,
+  styled,
+  theme,
+} from "@webstudio-is/design-system";
+import {
+  DiscordIcon,
+  GithubIcon,
+  TwitterIcon,
+  WebstudioLogoFlatIcon,
+  YoutubeIcon,
+} from "@webstudio-is/icons";
+import { useState, type ComponentProps } from "react";
+
+const StyledLink = styled("a", {
+  textDecoration: "none",
+  color: "white",
+  display: "inline-flex",
+  cursor: "default",
+});
+
+const PopoverItemButton = ({
+  href,
+  ...buttonProps
+}: ComponentProps<typeof Button> & { href?: string }) => (
+  <StyledLink href={href} target="_blank" referrerPolicy="no-referrer">
+    <Button
+      css={{
```

Comment:
instead of doing this, use flex around them to stretch

Reply:
then you don't need this component at all

Reply:
Ok. So should I remove the button component completely? What about the color variants? Will it apply on any component?
Or do you mean the link component?

Reply:
> instead of doing this, use flex around them to stretch

It doesn't seem to work.

Here's how I used it.
```
<StyledLink href={href} target="_blank" referrerPolicy="no-referrer">
    <Flex>
      <Button {...buttonProps} />
    </Flex>
  </StyledLink>
```

Reply:
You can also just try it out in the browser and let me know.